MainWindowController.swift

```diff
@@ -0,0 +1,44 @@
+//
+//  MainWindowController.swift
+//  Antidote
+//
+//  Created by Dmytro Vorobiov on 02/12/16.
+//  Copyright © 2016 Dmytro Vorobiov. All rights reserved.
+//
+
+import Cocoa
+
+class MainWindowController: NSWindowController {
```

Comment:
This is main window controller, it holds split controller, which has contacts and chats inside.